posts/inside-rust/2020-10-08-regression-labels.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Marking issues as regressions"
+author: Camelid
+description: "Now anyone can mark issues as regressions!"
+team: the release team <https://www.rust-lang.org/governance/teams/operations#release>
+---
+
+If you've contributed to Rust before, you have likely used a command like this
+to [set labels][rustbot-labeling-docs]:
+
+> @**rustbot** modify labels: A-typesystem C-enhancement
+
+Setting labels on issues and pull requests helps us keep track of and filter
+issues. There are many kinds of labels you can set to mark an issue
+as pertinent to a particular team, related to a part of the compiler, or
+what kind of bug it is (a hang, a crash, etc.).
+
+We also have labels that help us keep track of regressions so that we can
+prioritize them and fix them quickly if they are severe. We have three
+regression labels, each representing when the regression occurred:
+
+* regression-from-stable-to-nightly
+* regression-from-stable-to-beta
+* regression-from-stable-to-stable
```

Comment:
```suggestion
* `regression-from-stable-to-nightly`
* `regression-from-stable-to-beta`
* `regression-from-stable-to-stable`
```

Reply:
Hmm, I actually think it's easier to read as plain text:

![image](https://user-images.githubusercontent.com/37223377/95507837-e4827080-0966-11eb-8a58-93de3cfb79c0.png)

---

![image](https://user-images.githubusercontent.com/37223377/95507806-d6cceb00-0966-11eb-91f2-586c8ab3af70.png)

Reply:
Imo a code block is better

Reply:
Alright, I still feel mixed, but both of you seem to prefer inline code style so I'll apply it :)